src/main/java/seedu/address/logic/Logic.java

```diff
@@ -47,4 +47,14 @@ public interface Logic {
      * Set the user prefs' GUI settings.
      */
     void setGuiSettings(GuiSettings guiSettings);
```

Comment:
![telegram-cloud-photo-size-5-6323385397220064551-y](https://github.com/AY2324S2-CS2103T-W12-1/tp/assets/66112309/3a27613d-fe44-4a6f-8b4d-64ea29dc3459)

Should we reset the guy height and width to fit the double columns for client details?